tests/tests.bats

```diff
@@ -89,6 +89,14 @@ actual_matches_expected() {
     [[ "$status" -eq 1 ]]
 }
 
+@test "with_debug test" {
+    TEST="with_debug"
+    TEST_DIR="$DATA_DIR/$TEST"
+    run "$RUN_SCRIPT" "$TEST" "$TEST_DIR" "$TEST_DIR"
+
+    [[ "$status" -eq 0 ]]
```

Comment:
`((` ought to be used for numeric tests.

Reply:
true, but "wontfix"